src/BuildResult.php

```diff
@@ -40,4 +44,36 @@ public function getErrors(): array
     {
         return $this->errors;
     }
+
+    public function getMetas(): Metas
+    {
+        return $this->builder->getMetas();
+    }
+
+    /**
+     * Returns the "master document": the first file whose toctree is parsed.
+     *
+     * Unless customized, this is "index" (i.e. file index.rst).
+     *
+     * @return string
```

Comment:
Could removed as the PHP signature has the same. Not sure what's the policy in this repo though.

Reply:
Yes, thank you - my editor added that, it was not my intention.